packages/utils/upgrade/resources/codemods/5.0.0/utils-public-interface.code.ts

```diff
@@ -0,0 +1,315 @@
+import { Transform, JSCodeshift, Collection } from 'jscodeshift';
+
+/*
+
+This codemod transforms @strapi/utils imports to change method calls to math the new public interface.
+It will also warn about removed functions to avoid breaking user code.
```

Comment:
aren't warn message currently silenced by the upgrade tool?

Reply:
ha maybe I tried it directly via jscodeshift. 

How hard would it be to allow displaying all warnings at the end or sth ?

Reply:
Everything's possible, but that's not implemented for now.

I'll take a look next week

Reply:
I'll leave them in the PR then